packages/design-system/hooks/useLogoOverrides/index.ts

```diff
@@ -0,0 +1,49 @@
+'use client'
+
+import { ServerSideRenderingOption, getCookie, setCookie } from '@baseapp-frontend/utils'
+
+import { atom, useAtom } from 'jotai'
+
+import { DEFAULT_LOGO_KEY, DEFAULT_LOGO_SETTINGS } from './constants'
+import { LogoOverrides } from './types'
+
+export const getLogoOverridesFromCookie = ({ noSSR = true }: ServerSideRenderingOption = {}) => {
+  const settings = getCookie<LogoOverrides>(DEFAULT_LOGO_KEY, { noSSR }) ?? DEFAULT_LOGO_SETTINGS
+
+  return settings
+}
+
+const initialLogos = getLogoOverridesFromCookie()
+
+const settingsAtom = atom<LogoOverrides>(initialLogos)
+
+/**
+ * By using `useLogoOverrides` with the `noSSR` option set to `false`, causes Next.js to dynamically render the affected pages, instead of statically rendering them.
+ */
+const useLogoOverrides = ({ noSSR = true }: ServerSideRenderingOption = {}) => {
+  const [settings, setSettings] = useAtom(settingsAtom)
```

Comment:
You could use the atomWithStorage here to solve the persistence issue mentioned before, but I don't know how others feel about adding another state management library just for this. Personally I think it's fine, since jotai it's supposed to have a different use case than zustand.

Reply:
I think we can go with a simpler solution (use local storage at the template level) since this feature was made to allow the sales team to demo BA with clients. As it should be built now, it's not something our regular clients will use. `jotai` is being used on the `useUISettings` because we needed to be able to optionally get the initial state from a cookie or not

Reply:
In the `useUISettings` we used cookies to store the theme information so we can get it from the client and from the server (and by doing so, we avoid glitches from server to client). In this case, I wouldn't really bother using local storage (client-only) and have this small glitch, since it's a feature for demonstrations

Reply:
Ok, if we are fine with not persisting thins along pages, it's fine by me to not bother with the localstorage.